pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies = [
     "requests==2.31.0",
     "google-cloud-storage==2.10.0",
     "junitparser==3.1.0",
-    "pydantic==1.10.9",
+    "pydantic==2.1.1",
```

Comment:
@adriengentil I think for now we'll have to exclude pydantic. the migration to v2 will require some changes

Reply:
also cc @eifrach

Reply:
show i allow `1.*.*` versions?
or ignore the entire package?

Reply:
you can pin it to ``1.*.*`` with the groups mechanism?
if so, it's preferred. if not, just exclude it

Reply:
created #227